### PR TITLE
Bugfix/#51 fix description not being set with remote feature repository

### LINKED ISSUE #53: Feature toggle description is not set with remote feature repository
The description logic added through #51 does not actually work when using a remote feature repository at the moment.

### PR DESCRIPTION
**Pull request by KevinVanSonsbeek**
Fixes #53 

Todo:

- [x] Add description to ListController
- [x] Add description to ViewController

### CODE REVIEW

**Comment by KevinVanSonsbeek** [comment]
Do you have time to review these fixes? @nusje2000

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -72,17 +72,13 @@ function ($v) {
...
-            ];
-        }
-
-        return [null];
+        return [
+            'nusje2000/feature-toggle-bundle',
+            $version,
+        ];
     }
 }
```
Comment (nusje2000) [inline src/DependencyInjection/Configuration.php:84]: Since the arguments are not conditional anymore, we can add them to the call directly.

**Review by nusje2000 — CHANGES_REQUESTED** [review]

**Comment by nusje2000** [comment]
@KevinVanSonsbeek one minor change request, but looks good 👍

File: src/DependencyInjection/Configuration.php (hunk 1 above)
Comment (KevinVanSonsbeek) [reply src/DependencyInjection/Configuration.php]: I've added them directly to the call now.

**Comment by KevinVanSonsbeek** [comment]
> @KevinVanSonsbeek one minor change request, but looks good +1

Would you have time to re-review the PR?

**Comment by KevinVanSonsbeek** [comment]
Would you have time to re-review this MR in the coming week? @nusje2000

**Comment by nusje2000** [comment]
Thanks for your work @KevinVanSonsbeek 🚀🚀🚀

**Comment by KevinVanSonsbeek** [comment]
Could you make a 4.0.1 tag for this fix? @nusje2000

**Comment by nusje2000** [comment]
@KevinVanSonsbeek https://github.com/nusje2000/feature-toggle-bundle/releases/tag/v4.0.1

**Pull request merged**